code/modules/mob/mob_helpers.dm

```diff
@@ -99,12 +99,12 @@
 				newletter = "...huuuhhh..."
 			else if(newletter == ".")
 				newletter = " *BURP*."
-		switch(rand(1, 20))
+		switch(prob(10) ? rand(1, 3) : 0)
```

Comment:
```suggestion
		switch(prob(15) ? rand(1, 3) : 0)
```
Unless my math is wrong, 3 valid numbers in 20 is 15% that it'll be a valid number.

Reply:
ironically i had that and then changed it to 10 percent